podcasts/PodcastListViewController.swift

```diff
@@ -219,10 +219,7 @@ class PodcastListViewController: PCViewController, UIGestureRecognizerDelegate,
 
     @objc private func createFolderTapped(_ sender: UIBarButtonItem) {
         if FeatureFlag.onboardingUpdates {
-            let hostingController = PCHostingController(rootView: PlusLandingView(dismissAction: {
-                self.dismiss(animated: true)
-            }))
-            present(hostingController, animated: true, completion: nil)
+            self.present(LoginCoordinator.make(), animated: true)
```

Comment:
A suggestion:

I think we can have 3 different parts:

1. The `LoginCoordinator` would just handle the taps and routing
2. You can create a `LoginNavigationController` in which you configure the navigation bar (removing that from the view)
3. Keep the view as just the view